apps/web/lib/web/controllers/github_webhook_controller.ex

```diff
@@ -19,39 +22,54 @@ defmodule Web.GitHubWebhookController do
   end
 
   def process_issue(%{"issue" => issue, "repository" => %{"html_url" => url, "name" => name}}) do
-    {:ok, project} = Projects.get_or_insert(%{url: url, name: name})
-
-    issue
-    |> attributes(project)
-    |> Opportunities.insert_or_update
+    case Projects.get(%{url: url}) do
+      nil -> Logger.info("Received issue payload for untracked project: #{name} #{url}")
+      project ->
+        issue
+        |> attributes(project)
+        |> Opportunities.insert_or_update
+    end
   end
 
   defp attributes(issue, %{id: project_id}) do
     %{
       closed_at: issue["closed_at"],
-      level:  level_from_labels(issue),
+      level:  label_mapping(issue, level_mapping()),
       project_id: project_id,
       title: issue["title"],
+      type:  label_mapping(issue, type_mapping()),
       url: issue["html_url"]
     }
   end
 
-  defp label_mapping, do: Application.get_env(:web, :level_label_mapping)
+  defp authorize(%{params: %{"api_token" => api_token}} = conn, _opts) do
```

Comment:
Super simple authentication to start.  We may want to get fancy with this later but it should do the trick for now.